src/kili/entrypoints/mutations/label/__init__.py

```diff
@@ -206,6 +211,9 @@ def append_labels(  # pylint: disable=dangerous-default-value
             project_id: Identifier of the project.
             asset_external_id_array: list of asset external ids to append labels on.
             disable_tqdm: Disable tqdm progress bar.
+            overwrite: when uploading predictions or inference labels, if True,
+                it will overwrite existing predictions and inference labels
+                with the same model name on the targeted assets.
```

Comment:
same comments as above

also, I don't understand the "it will overwrite existing predictions and inference labels".

Should we really overwrite both prediction and inference labels if we call kili.append_labels(label_type="prediction")?

I think we should only delete existing labels of the same type

If a user wants to delete both, he can call kili.append_labels a second time with the other label type

wdyt?

Reply:
yes it is the current behavior, it is the docstrings that doesn't explain it well indeed